infracost/1.0.0/README.md

```diff
@@ -0,0 +1,64 @@
+# Infracost Integration with Keptn
+
+![infracost running in keptn](./assets/infracost-keptn.png)
+
+This integration uses the [job executor service](https://github.com/keptn-contrib/job-executor-service).
+
+## Prerequisites
+
+You will need:
+
+- An infracost API token (see [get an API token](https://www.infracost.io/docs/#2-get-api-key))
+
+## How it Works
+The Job executor service is configured to listen for a `sh.keptn.event.{task}.triggered` event (if following the example files, this is `sh.keptn.event.checkcost.triggered`).
+
+The job executor:
+1. Creates a new Kubernetes job
+2. Copies in the Terrafrom plan.json file from the Keptn git repo at `/files/plan.json` into the container at `/keptn/files/plan.json`.
+3. Reads the `infracost-details` secret and sets the `INFRACOST_API_TOKEN` environment variable
+4. Sends an `sh.keptn.event.checkcost.started` event to Keptn
+5. Runs `infracost breakdown --path /keptn/files/plan.json --format table`
+6. Sends an `sh.keptn.event.checkcost.finished` event to Keptn, with the tabular output
+
+## Install Job Executor Service
+Follow the [Job executor service install instructions](https://github.com/keptn-contrib/job-executor-service#quickstart) to install on a cluster (either local to Keptn control plane components or on a different cluster).
+
+Ensure the Job Executor Service is installed listening for the `sh.keptn.event.taskname.triggered` event that corresponds to your Shipyard file (see below).
+
+Below, the task is `checkcost` so JES should listen for `sh.keptn.event.checkcost.triggered`.

```

Comment:
Does this add anything that the line above did not already mention?

Reply:
IMO people scan instructions and miss things so I'm going to opt to leave it in :)